apps/web/vibes/soul/form/swatch-radio-group/index.tsx

```diff
@@ -30,16 +30,18 @@ export const SwatchRadioGroup = React.forwardRef<
     options: SwatchOption[];
     errors?: string[];
   }
->(({ id, label, options, errors, className, ...rest }, ref) => {
+>(({ label, options, errors, className, ...rest }, ref) => {
+  const id = React.useId();
+
   return (
     <div className={clsx('space-y-2', className)}>
       {label !== undefined && label !== '' && <Label htmlFor={id}>{label}</Label>}
```

Comment:
We should use `id`, not `htmlFor` when using `aria-labelledby` (this seems to be needed across a couple of components)

Reply:
Done.